src/components/layout/livechat.tsx

```diff
@@ -1,13 +1,20 @@
 import React, { useState } from 'react'
-import PropTypes from 'prop-types'
 import styled, { css } from 'styled-components'
 import { useLivechat } from 'components/hooks/use-livechat'
 import LiveChatIC from 'images/svg/layout/livechat.svg'
 import LiveChatHover from 'images/svg/layout/livechat-hover.svg'
 import device from 'themes/device'
 import { DerivStore } from 'store'
 
-const StyledLiveChat = styled.div`
+interface LiveChatProps {
+    is_banner_shown: boolean
+}
+
+interface StyledLiveChatProps extends LiveChatProps {
+    is_eu_country: boolean
+}
```

Comment:
We should ideally reach for `type` first until we need an `interface`. You could do either of this approach below.

```tsx
type LiveChatProps = {
    is_banner_shown: boolean
}

type StyledLiveChatProps = LiveChatProps & {
    is_eu_country: boolean
}
```

```tsx
type LiveChatProps = {
    is_banner_shown: boolean
}

interface StyledLiveChatProps extends LiveChatProps {
    is_eu_country: boolean
}
```

Reply:
Same for the rest below.